### DISCUSSION TITLE
Aksyon TV International

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#652] [Q&A]
**Channel Name:** AksyonTV International
**Channel ID:** AksyonTVInternational.ph
**Country:** Philippines
**Language:** Tagalog
**External Link (optional):** xxx
**Notes (optional):** Sister channel of Kapatid Channel

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19652
